editor/src/components/import-modal.js

```diff
@@ -0,0 +1,270 @@
+/* eslint-disable no-undef */
+/* eslint-disable camelcase */
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useState, useEffect } from '@wordpress/element';
+import { Modal, Button } from '@wordpress/components';
+import { parse } from '@wordpress/blocks';
+import { close } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+import Header from './header';
+import Content from './content';
+import PreviewFrame from '../../../assets/src/Components/CloudLibrary/PreviewFrame';
+import { importTemplate } from '../data/templates-cloud';
+
+const { omit } = lodash;
```

Comment:
We can dump lodash as 6.4 won't enqueue it by default, as similar issue being fixed in this PR: https://github.com/Codeinwp/templates-patterns-collection/pull/310